ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.html

```diff
@@ -23,8 +27,8 @@
             <span *ngIf="col.srTitle" class="sr-only">{{col.srTitle}}</span>
             <span *ngIf="col.title">{{col.title}}</span>
         </th>
-        <!-- Actions -->
-        <th *ngIf="actions">
+        <!-- Actions (right) -->
```

Comment:
Can we have a couple of unit tests that checks the action be the first column and last if is left or right?